docs/tutorials/star-field.md

```diff
@@ -255,4 +490,4 @@ game.onUpdate(function () {
         projectile.setFlag(SpriteFlag.Ghost, true)
     }
 })
-```
```

Comment:
Looks like the code got appended here instead of deleting the parts you edited, I'm about to have dinner but I'll see what got switched up in a few. Otherwise edits look great!